sal_interfaces/SALSubsystems.xml

```diff
@@ -163,6 +163,52 @@
   <Configurable>No</Configurable>
 </SALSubsystem>
 
+<SALSubsystem>
+  <Name>CCOODS</Name>
+  <Description>ComCam OODS</Description>
+  <Enumeration></Enumeration>
+  <Generics>yes</Generics>
+  <Version></Version>
+  <Author></Author>
+  <ActiveDevelopers>Steve Pietrowicz</ActiveDevelopers>
+  <PrincipalCSCOwner>Michael Reuter</PrincipalCSCOwner>
+  <Github>https://github.com/lsst-dm/ctrl_oods</Github>
+  <JenkinsTestResults>NA</JenkinsTestResults>
+  <LSSTPoC>Michael Reuter</LSSTPoC>
+  <CSCDocs>NA</CSCDocs>
+  <ProductOwner>Michael Reuter</ProductOwner>
+  <RelatedDocuments>Don't know but there is</RelatedDocuments>
+  <SoftwareLanguage>Python</SoftwareLanguage>
+  <RuntimeLanguages>CPP,LabVIEW,Java,PyDDS,Python</RuntimeLanguages>
```

Comment:
If you are using SalObj for this, your runtime is PyDDS.

Reply:
Fixed